app/domain/rotation/rotators/postgresql/password.rb

```diff
@@ -103,6 +96,29 @@ def username_id
           end
         end
 
+        class RotatedDb
```

Comment:
I'm not loving this class name because it doesn't clearly communicate what it is responsible for. 

`DbPasswordUpdater`, `DbUpdater`, or if you wanted to keep it more generic, even just `DbConnection` or `Database`. 

Any of those better communicate, at least to me, what I expect an instance of the class to represent.

Reply:
Fair enough, but `DbPasswordUpdater` and `DbUpdater` are [out of the question](http://www.carlopescio.com/2011/04/your-coding-conventions-are-hurting-you.html).  (I have about 5 more links on this subect if your interested :) ).

I'll go ahead and change it to `Database` though.